tests/test_protocol.py

```diff
@@ -283,9 +284,9 @@ def test_decode_fetch_response_partial():
             b"ar",  # Value (truncated)
         ]
     )
-    resp = FetchResponse[0].decode(io.BytesIO(encoded))
-    assert len(resp.topics) == 1
-    topic, partitions = resp.topics[0]
+    resp = FetchResponse_v0.decode(io.BytesIO(encoded))
+    assert len(resp.topics) == 1  # type: ignore[attr-defined]
+    topic, partitions = resp.topics[0]  # type: ignore[attr-defined]
```

Comment:
We should rather fix this problem in response class

Reply:
I see these ways:
1. Use annotations on struct classes and generate __init__/attributes from these annotations (maybe use descriptor and dataclass/dataclass_transform)
2. Add annotations to class from schemas
3. Write mypy plugin like the existing mypy plugin for dataclasses